### PR TITLE
VAE updates - Round 2

### PR DESCRIPTION
**Pull request by nmdickso**
So, the main thing is that for the (admittedly edge-case) `copernicus` RNN, the reparametrization is only done for the "first pass" of the network, during encoding, while the recurrent time evolution requires a simpler loss function.

So, rather than defining and working that all out myself in `copernicus`, I think it's much easier and cleaner to simply re-juggle the loss function initialization a tiny bit. I've walked back the changes to the `hyperparameters` and now the leading loss function is simply defined explicitly in there, then initialized and stored as `self.leadingLoss` in `__init__`, and then used in @keyblaede's VAE loss function. This way, when I don't need the full VAE loss, I can simply use the leading loss function instead.

I'll be honest, I never really understood changing the leading loss to a parsed string, it seemed a little convoluted. That's pretty much the only thing changed here, and it makes my life easier this way. If you've got a good reason against what I've done here, let me know and I can look at it again.

### CODE REVIEW

**Comment by nmdickso** [comment]
Oh, I also took out the `verbose` keyword from the forward function cause it wasn't being used and it made a couple lines >80 characters.

**Comment by httpjesse** [comment]
All looks good to me.

**Review by httpjesse — APPROVED** [review]
